README.md

```diff
@@ -131,6 +131,14 @@ $ cat .gitattributes
 *.rb linguist-language=Java
 ```
 
+Note that, when using Linguist locally, `.gitattributes` must be committed again each time it is modified for Linguist to take it into account. Also note that you might need to use wildcards to match files deeper than top-level ones.
```

Comment:
Maybe amend this so it reads:

> **Note:** When run locally, Linguist will ignore uncommitted changes to `.gitattributes`. Remember to commit each modification before rerunning `linguist`.

Reply:
And yes, make sure you get rid of this fluff:

>  Also note that you might need to use wildcards to match files deeper than top-level ones.

Reply:
Since `When testing with a local installation of Linguist, take note that the added attributes will not take effect until the .gitattributes file is commited to your repository.` has already been added with https://github.com/github/linguist/pull/4115, I should simply remove this paragraph :slightly_smiling_face:

Reply:
Ah. Erm... yes.

So, uh, I guess that reduces this PR down to amending the fenced code-block with `console`. 😅